CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Fix: NDK integration was being disabled by a typo
 - Fix: missing toList for debug meta #192
 - Enhancement: NavigationObserver to record Breadcrumbs for navigation events #197
+- Fix: close the isolateErrorIntegration receivePort on Sentry.close()
```

Comment:
I'd change to something more generic now, which is a system to dispose integrations

Reply:
✅ `add an integration closing system`?